plugin/src/main/java/com/iridium/iridiumskyblock/database/Island.java

```diff
@@ -166,7 +166,7 @@ public double getValueOf(XMaterial material) {
      * Must be lower than the distance between Islands.
      */
     public int getSize() {
-        return IridiumSkyblock.getInstance().getConfiguration().distance - 1;
+        return IridiumSkyblock.getInstance().getUpgrades().sizeUpgrade.upgrades.get(IridiumSkyblock.getInstance().getIslandManager().getIslandUpgrade(this, "size").getLevel()).size;
```

Comment:
What is this line? o.O

Reply:
wdym?

Reply:
It's pretty long and confusing

Reply:
This better?

Reply:
https://github.com/Iridium-Development/IridiumSkyblock/pull/59/files#diff-12abb872602bfd952f244bf7cca114d9cd538e2d5b5e4ce2c3e4dd48f0068eebR169

Reply:
I guess